docs/ops/reduction/ReduceMean_1.md

```diff
@@ -33,11 +33,11 @@
 
 **Detailed Description**
 
-Each element in the output is the result of reduction with finding the arithmetic mean operation along dimensions specified by the 2nd input:
+Each element in the output is the result of arithmetic mean reduction operation along dimensions specified by the 2nd input:
 
     output[i0, i1, ..., iN] = mean[j0,..., jN](x[j0, ..., jN]))
 
-Where indices i0, ..., iN run through all valid indices for the 1st input and finding the arithmetic mean `mean[j0, ..., jN]` have `jk = ik` for those dimensions `k` that are not in the set of indices specified by the 2nd input of the operation. 
+Where indices i0, ..., iN run through all valid indices for the 1st input and finding the arithmetic mean `mean[j0, ..., jN]`.`jk = ik` for those dimensions `k` that are not in the set of indices specified by the 2nd input of the operation. 
```

Comment:
Why did you change `have` to `.`?

Reply:
I misunderstood the sentence. I will undo it